jdk/make/CompileLaunchers.gmk

```diff
@@ -93,6 +93,18 @@ define SetupLauncher
   endif
 
   $1_LDFLAGS := $3
+  
+  ifeq ($(OPENJDK_TARGET_OS), linux)
+  # Set the image base address for zLinux 64 to 0x60000 for launchers,
+  # to make compressedRefsShift to allow compressedRefsShift to be 0 when -Xmx is set to 2040m or more.
+  # / RTC PR 100052
+    ifeq ($(OPENJDK_TARGET_CPU), s390x)
+      ifeq ($(OPENJDK_TARGET_CPU_BITS), 64)
+        $1_LDFLAGS += -Wl,-Ttext-segment=0x60000
+      endif
```

Comment:
I think you have the 32-bit options for this as well, might as well add it, in case it's needed in the future.

Reply:
In that case, would it be worth just removing the check for 64 bits altogether?

Reply:
I thought the 32-bit and 64-bit solutions used different options.

Reply:
Maybe I've confused this with AIX

Reply:
Oh are you referencing this comment? https://github.com/eclipse/openj9/issues/7115#issuecomment-544535974

If so, I found that it is a AIX only option. This fix should work for 32 bit too but I don't have access to an AIX box to test this theory